src/index.js

```diff
@@ -686,14 +686,48 @@ PubSub.prototype.request = function(config, callback) {
 
   if (!gaxClient) {
     // Lazily instantiate client.
-    gaxClient = v1(this.options)[config.client](this.options);
+    if (config.client === 'publisherClient') {
+      gaxClient = new v1.PublisherClient(this.options);
+    }
+    else if (config.client == 'subscriberClient') {
+      gaxClient = new v1.SubscriberClient(this.options);
+    }
+    else {
+      throw new Error("Client is unknown: " + config.client);
+    }
+    var scopes = gaxClient.constructor.scopes;
+    extend(this.options, { scopes });
```

Comment:
Might as well just assign `this.options.scopes = gaxClient.constructor.scopes`.

Although, it might be better to do this the way @lukesneeringer did for Logging: https://github.com/googleapis/nodejs-logging/blob/bf3936c7979b4b4d6a55f48604b0c266bc58fc90/src/index.js#L116-L130. It combines all of the scopes from all required classes.

Reply:
Now getting scopes from all v1 classes.